src/components/BackButton.js

```diff
@@ -34,7 +34,7 @@ const BackButton = (props) => {
 
 	const handleClick = () => {
 		if (props.warn) {
-			alert=true;
+			setAlert=true;
```

Comment:
Incorrect. 
When we are using functional components we don't set state by ```=```. **Look at the differences**
 
Functional component 
```
const Backbutton = ( ) => {
      const [alert, setAlert] = React.useState(false)
    
         setAlert(true)
    }
```
Notice how I don't need to use ```this.setState```, we just use the 2nd item in that array as a function. 

Class Component  
```
class BackButton extends React.Component{
    constructor(props){
        super(props);
        this.state = {
            alert : false
        }
    }
    this.setState({alert: true})
}

```

Notice how this is so much more code for the same thing.

When working with React states you will always either of the two.

❗ **Set the state in the state in the BackButton component with the function component way** ❗ 

source: https://reactjs.org/docs/state-and-lifecycle.html